frontend/src/routes/FacilityBooking/CreateBooking/index.tsx

```diff
@@ -173,19 +177,20 @@ export default function CreateBooking() {
       ) : (
         <Form onSubmit={onSubmit}>
           <Controller
+            name="eventName"
             control={control}
-            render={({ onChange, value }) => (
+            render={({ onChange, name }) => (
               <InputField
-                hasError={value === ''}
+                name={name}
+                hasError={!!errors.eventName}
                 title="Event Name"
                 placeholder="Event Name"
-                value={value}
```

Comment:
does this work without passing value in?

Reply:
ecf02eb

Reply:
this works? there is no need to pass value into the input field?

Reply:
yeah it works when i console.log(watch())